CMakeLists.txt

```
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.21 FATAL_ERROR)
 cmake_policy(VERSION 3.21.1...3.27)
 
 # Create project
-project(locality_aware LANGUAGES C CXX)
+project(locality_aware VERSION 0.1.0 LANGUAGES C CXX)
```

```suggestion
project(locality_aware VERSION 2.0.0 LANGUAGES C CXX)
```
Or whatever to match the next version we want to release (current version is 1.0.0)